app/Http/Controllers/AccountController.php

```diff
@@ -74,6 +79,12 @@ public function update(Request $request)
         $user->allow_profile_contact = $request->get('allow_profile_contact');
         $user->profile_intro = $request->get('profile_intro');
         $user->profile_slug = $request->get('profile_slug');
+
+        if ($request->file('profile_picture')) {
+            $image = $request->file('profile_picture');
+            Event::fire(new ProfilePictureUpdated($user, $image->getRealPath(), $image->getClientOriginalExtension()));
```

Comment:
Are we sure that handling file uploads in a queued event will work? I don't know how long uploaded files stick around...